frontend/javascripts/libs/input.ts

```diff
@@ -67,28 +67,70 @@ function shouldIgnore(event: KeyboardEvent, key: KeyboardKey) {
 // This keyboard hook directly passes a keycombo and callback
 // to the underlying KeyboadJS library to do its dirty work.
 // Pressing a button will only fire an event once.
+const EXTENDED_COMMAND_KEYS = "ctrl + k";
+const EXTENDED_COMMAND_DURATION = 3000;
 export class InputKeyboardNoLoop {
   bindings: Array<KeyboardBindingPress> = [];
   isStarted: boolean = true;
   supportInputElements: boolean = false;
+  extendedBindings: Array<KeyboardBindingPress> = [];
+  cancelExtendedModeTimeoutId: ReturnType<typeof setTimeout> | null = null;
 
   constructor(
     initialBindings: BindingMap<KeyboardHandler>,
     options?: {
       supportInputElements?: boolean;
     },
+    extendedCommands?: BindingMap<KeyboardHandler>,
   ) {
     if (options) {
       this.supportInputElements = options.supportInputElements || this.supportInputElements;
     }
 
+    if (extendedCommands != null && initialBindings[EXTENDED_COMMAND_KEYS] != null) {
+      console.warn(
+        `Extended commands are enabled, but the keybinding for it is already in use. Please change the keybinding for '${EXTENDED_COMMAND_KEYS}'.`,
+      );
+    }
+
+    if (extendedCommands) {
+      this.attach(EXTENDED_COMMAND_KEYS, this.toggleExtendedMode);
+      // Add empty callback in extended mode to deactivate the extended mode via the same EXTENDED_COMMAND_KEYS.
+      this.attach(EXTENDED_COMMAND_KEYS, _.noop, true);
```

Comment:
No, unfortunatley, this does not work for me :/ 
I'll try a bit whether I can fix this as github seemed to have managed it. Maybe I can find a workaround.

Reply:
I just pushed a fix that works on Firefox. I also checked for chrome, but my chrome installation is somewhat broken. Thus it would be best, if you could check this again